libraries/botbuilder-ai/src/localeConverter.ts

```diff
@@ -0,0 +1,212 @@
+/**
+ * @module botbuilder-ai
+ */
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { Middleware } from 'botbuilder';
+import { Activity, ResourceResponse } from 'botbuilder';
+import * as LanguageMap from './languageMap';
+import * as DateTimeRecoginizers from '@microsoft/recognizers-text-date-time';
```

Comment:
>@microsoft/recognizers-text-date-time [](start = 39, length = 37)

you should add a dependency to package.json for this.

Reply:
The dependency is added to the global package.json, do you mean it should only be included in the botbuilder-ai dependencies?